FordLabs/PeopleMover
# 126 hide notes icon placeholder borders in view only

## Issue
Resolves #126 Hide Notes Icon & Placeholder Borders in View Only

## What was done
- [x] Removed notes icon from readonly view
- [x] Removed placeholder border on readonly view
## How to test
1. Visit space you only have read access to
2. Confirm that users with notes don't have icons
3. Confirm placeholders don't have the border color of their role.
